test/pom.xml

```diff
@@ -81,7 +81,7 @@ THE SOFTWARE.
     <dependency>
       <groupId>${project.groupId}</groupId>
       <artifactId>jenkins-test-harness</artifactId>
-      <version>1589.vc23fca066d5c</version>
+      <version>1626.v46b0925e70db</version>
```

Comment:
are the test failures related to updating this?

Reply:
any idea why dependabot isn't updating this?

```
updater | INFO <job_213087289> Checking if org.jenkins-ci.main:jenkins-test-harness 1589.vc23fca066d5c needs updating
  proxy | 2021/09/26 19:10:19 [146] GET https://repo.jenkins-ci.org:443/public/org/jenkins-ci/jenkins/1.67/jenkins-1.67.pom
  proxy | 2021/09/26 19:10:19 [146] 200 https://repo.jenkins-ci.org:443/public/org/jenkins-ci/jenkins/1.67/jenkins-1.67.pom
  proxy | 2021/09/26 19:10:19 [148] GET https://repo.jenkins-ci.org:443/public/org/jenkins-ci/main/jenkins-test-harness/maven-metadata.xml
  proxy | 2021/09/26 19:10:19 [148] 200 https://repo.jenkins-ci.org:443/public/org/jenkins-ci/main/jenkins-test-harness/maven-metadata.xml
  proxy | 2021/09/26 19:10:19 [150] GET https://repo.maven.apache.org:443/maven2/org/jenkins-ci/main/jenkins-test-harness/maven-metadata.xml
  proxy | 2021/09/26 19:10:19 [150] 404 https://repo.maven.apache.org:443/maven2/org/jenkins-ci/main/jenkins-test-harness/maven-metadata.xml
  proxy | 2021/09/26 19:10:19 [152] HEAD https://repo.jenkins-ci.org:443/public/org/jenkins-ci/main/jenkins-test-harness/1589.vc23fca066d5c/jenkins-test-harness-1589.vc23fca066d5c.jar
  proxy | 2021/09/26 19:10:19 [152] 200 https://repo.jenkins-ci.org:443/public/org/jenkins-ci/main/jenkins-test-harness/1589.vc23fca066d5c/jenkins-test-harness-1589.vc23fca066d5c.jar
updater | INFO <job_213087289> Latest version is 1589.vc23fca066d5c
```

Reply:
No idea offhand.